sdks/typescript/package.json

```diff
@@ -21,16 +21,20 @@
     "test": "mocha dist/test",
     "codecovTest": "istanbul cover mocha dist/test --reporter lcovonly -- -R spec && codecov",
     "prettier": "prettier --write src/",
-    "prettier-check": "prettier --check src/"
+    "prettier-check": "prettier --check src/",
+    "lint": "eslint . --ext .ts"
   },
   "dependencies": {
     "@grpc/grpc-js": "^1.4.6",
     "@protobuf-ts/grpc-transport": "^2.1.0",
     "@protobuf-ts/plugin": "^2.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.23.0",
```

Comment:
Should these be devDependencies rather than dependencies?

Reply:
Yes, they should. Good catch. Fixed. Thanks!